test/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "name" {
+  type        = string
+  description = "Name"
+}
+
+variable "region" {
```

Comment:
Region and profile can be set from env vars, consider whether you need these variables and explicit provider configuration.

Reply:
this worked, removed